example/build.gradle

```diff
@@ -2,31 +2,18 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 31
+    compileSdk 33
 
     defaultConfig {
         applicationId "com.github.appintro.example"
-        minSdkVersion 14
-        targetSdkVersion 31
-        versionCode 2
-        versionName "1.0.1"
```

Comment:
Why are we removing those?

Reply:
As we're not publishing this App (are we?).